db/migrate/20191220180548_devise_create_models.rb

```diff
@@ -1,44 +0,0 @@
-# frozen_string_literal: true
```

Comment:
why are you deleting the migration!? what if something fails on your server? you won't be able to roll back the changes you made to the db via rails

Reply:
I deleted because I had problems when I created the user model, the model table was unnecesary so I deleted. Yes it was a terrible solution